### ISSUE TITLE
[Question] What was the SFT baseline trained on?

### ISSUE THREAD

**Issue by jwong8314** [rookie-joe/PDA#2]
Hi, 

I'm looking to replicate the 20% pass rate achieved by fine-tuning on FormL4 (love the name by the way !!). 

Was that done by taking the user prompt to be "nl_problem" in train.json and taking ground truth to be "formal"? 

Were the explanations included in the training process also?

Justin

**Comment by rookie-joe** [comment]
Thank you for your interest in replicating the FormL4 fine-tuning results. You're correct about the 20% pass rate (i think u can make it much higher using [Qwen2.5-Math-7B](https://huggingface.co/Qwen/Qwen2.5-Math-7B)) - let me clarify the training setup.

The training process uses a completion model approach, which is simpler than conversational templates like alpaca. The input format is structured as follows:

```
Statement and proof in natural language:
# Statement:
{nl_problem}
# Proof:
{nl_proof}
Translate the statement and proof in natural language to Lean 4:
```

The corresponding training label is the formal Lean 4 code representation.

While you could experiment with conversational templates for comparison, this direct completion approach proved effective for our use case.
